packages/bridge/src/client.ts

```diff
@@ -0,0 +1,46 @@
+import { NetworkConfig } from "@arcadeum/provider"
+import { BigNumberish, providers } from "ethers"
+import { Bridge, Move } from "./bridges"
+import { flatten, safeSolve } from "./utils"
+
+
+export class BridgesClient {
+  public bridges: Bridge[]
+
+  constructor(bridges: Bridge[] = [], networks: NetworkConfig[] = []) {
+    this.bridges = bridges.map((bridge) => bridge.connect(networks))
+  }
+
+  connect(networks: NetworkConfig[]): BridgesClient {
+    this.bridges = this.bridges.map((bridge) => bridge.connect(networks))
+    return this
+  }
+
+  get(id: string): Bridge {
+    return this.bridges.find((b) => b.id() === id)
+  }
+
+  async optionsNative(from: NetworkConfig, to: NetworkConfig): Promise<Bridge[]> {
```

Comment:
I'm not sure what these `options` functions are. Would you mind explaining? It returns a list of available bridges for a given token standard?

Reply:
Yes, most likely a client would be integrating multiple bridges at the same time